docs/manifest.json

```diff
@@ -248,6 +248,12 @@
           "path": "./admin/automation.md",
           "icon_path": "./images/icons/plug.svg"
         },
+        {
+          "title": "Scaling Coder",
+          "description": "Reference architecture and load testing tools",
+          "icon_path": "./images/icons/scale.svg",
+          "path": "./admin/scale/index.md"
```

Comment:
nit: do we expect more pages or would `scale.md` be sufficient?

Reply:
Down the road, we may provide reference architectures! I'll just make it `scale.md` though